openlibrary/plugins/upstream/borrow.py

```diff
@@ -464,8 +467,8 @@ def get_all_loans():
     return get_all_store_values(type='/type/loan')
 
 
-def get_loans(user):
-    return lending.get_loans_of_user(user.key)
+def get_loans(user, use_cache=True):
```

Comment:
`use_cache=False` so that the default case is not to use cache and as we discover opportunities we can "opt-in" rather than "opt-out" (because this seems safer)